examples/torque_driven_ocp/pendulum_with_fatigue.py

```diff
@@ -0,0 +1,154 @@
+"""
```

Comment:
Is this file a copy of `examples/fatigue/pendulum`? If so, please remove this one. 
Also I doubt the "with_fatigue" in the name are necessary since they are in a fatigue folder